### ISSUE TITLE
Test status is not shown

### ISSUE THREAD

**Issue by lgoltz** [opengeospatial/ets-wms-client13#29]
ETS lists all the tests that need to execute but does not show the test status (pass/fail)

* Production (http://cite.opengeospatial.org/teamengine, ETS WMS  Client 1.3.0, v1.2, TEAM Engine 4.10)
![wms_client_test](https://user-images.githubusercontent.com/4108977/34785750-eeac9a54-f5e6-11e7-87a5-ace91fe935fd.png)

* On a in-house server hosting the same version as the production the test result never updates:
![wms_client_test_enterprise](https://user-images.githubusercontent.com/4108977/34786007-9fdb2034-f5e7-11e7-910f-7e53ac1d816b.png)

* After I stop the test execution, I get this result with no other information:
![wms_client_test_enterprise1](https://user-images.githubusercontent.com/4108977/34786076-de88ce12-f5e7-11e7-88c2-111294599e51.png)

(Original issue: https://github.com/opengeospatial/teamengine/issues/301)

**Comment by lgoltz** [comment]
@vinnakota Please ensure you use the correct URL in the client. This must be an URL like http://cite.opengeospatial.org/teamengine/monitor/3

Please try to send the following request manually (e.g via browser):  <URL_PROVIDED_BY_ETS>?REQUEST=GetCapabilities&SERVICE=WMS
The first test ("Perform at least one GetCapabilities request") should be marked as successful. Please report the result (the result page of the ETS) here.

**Comment by vinnakota** [comment]
@lgoltz In production (on http://cite.opengeospatial.org/teamengine ) the test engine does not give me any WMS URL to connect to. To repro this issue:
1. Log in to TeamEngine and create a new session
2. Select "Web Map Service (WMS) - Client - 1.3.0 [1.2] specification
3. Give this session a description
4. Click on "Start a new test session" button.
5. You will get a "Executing tests...." page and it never goes past this to give a WMS URL.
--------------------------------
As for the in house version of test engine, I get the URL and am able to connect to the URL using the client. Requests are sent correctly and the test engine does process the results. Problem is test engine  does not update the results as it processes the requests. Only after I stop the test execution does it display the summary of test results, but does not list individual test results.

The last time I executed these tests (I believe sometime in 2017), these tests were fine. Each test would update its status (pass/fail) as soon as the test engine processes a request.

**Comment by lgoltz** [comment]
@vinnakota Thanks for describing the steps in detail!

Production is running again.  I've observed this on a local running ETS. We have to made some investigations. 

The problem you describe with your local environment are already described https://github.com/opengeospatial/ets-wms-client13/issues/24. We are currently working on it.  The problem also occur on production.

**Comment by lgoltz** [comment]
A new version of the ETS WMS Client 1.3 was installed on beta with some bug fixes related to the described problem.
@vinnakota Can you recheck on [beta](http://cite.opengeospatial.org/te2/)?

**Comment by vinnakota** [comment]
@lgoltz the new version on Beta is much better. The test engine expects a request for cite:terrain layer that does not exist in the capabilities causing one test to fail.
another minor issue: the test engine seems to parse only the first layer in a GetMap request. Because of this I have to toggle each layer visibility individually for the parser to recognize the requests being sent.

**Comment by lgoltz** [comment]
@vinnakota Thanks for quick response! The problem with the layer cite:Terrain is already documented in #32. I've created  a new issue for the other one: #34
